app/src/main/java/net/gsantner/markor/util/AppSettings.java

```diff
@@ -771,14 +771,6 @@ public void setPasswordHasBeenSetOnce(boolean b) {
         setBool(R.string.pref_key__default_encryption_password_set_once, b);
     }
 
-    public boolean getNewFileDialogLastUsedEncryption() {
-        return getBool(R.string.pref_key__new_file_dialog_lastused_encryption, false);
-    }
-
-    public void setNewFileDialogLastUsedEncryption(boolean b) {
-        setBool(R.string.pref_key__new_file_dialog_lastused_encryption, b);
```

Comment:
We use the `.jenc` extension to detect encryption everywhere else. These options are redundant. We can just use the remembered extension.